.pipelines/templates/aks-setup.yaml

```diff
@@ -13,6 +13,7 @@ steps:
           -n ${AZURE_CLUSTER_NAME} \
           --enable-managed-identity \
           --no-ssh-key \
+          --kubernetes-version ${K8S_VERSION} \
```

Comment:
Where is this value being passed in?

Reply:
This is configured in the UI in the "variables" section: https://dev.azure.com/AzureContainerUpstream/OpenServiceMesh/_apps/hub/ms.vss-build-web.ci-designer-hub?pipelineId=264&nonce=MRVfuedDfRjzTif0g1tjfA%3D%3D&branch=main